assets/js/main.js

```diff
@@ -0,0 +1,29 @@
+const navbtn = document.querySelector(".nav-btn");
+const sidemenu = document.querySelector(".sidemenu");
+const closebtn = document.querySelector(".close-btn");
+
+navbtn.addEventListener("click", () => {
+    navbtn.classList.toggle("active");
+    sidemenu.classList.toggle("active");
+})
+
+closebtn.addEventListener("click", () => {
+    sidemenu.classList.toggle("active");
+})
+
+document.querySelectorAll(".nav-link").forEach(n => n.addEventListener("click", () => {
+    navbtn.classList.remove("active");
+    sidemenu.classList.remove("active");
+}))
```

Comment:
Codacy has a fix for the issue: [Replace `}))` with `··})⏎);`](https://app.codacy.com/gh/aerial978/blog/pullRequest?prid=9584604)
```suggestion
  })
```